3/3scale-istio-adapter/Dockerfiles/build.sh

```diff
@@ -0,0 +1,58 @@
+# ----------------------------------------------------------------------------
+# Package       : 3scale-istio-adapter 
+# Version       : code-cleanups
+# Source repo   : https://github.com/3scale/3scale-istio-adapter
+# Tested on     : RHEL_8.2
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Krishna Harsha Voora
+#
+# Disclaimer: This script has been tested in non-root mode on given platform using
+#             the mentioned version of the package. It may not work as expected 
+#             with newer versions of the package and/or distribution.
+#             In such case, please contact "Maintainer" of this script.
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+# Download/Setup Golang 1.11
+cd /usr/local/
+wget https://golang.org/dl/go1.11.13.linux-ppc64le.tar.gz
+tar -zxf go1.11.13.linux-ppc64le.tar.gz
+rm -f go1.11.13.linux-ppc64le.tar.gz
+
+
+# Update PATH Variables
+export GOPATH=$HOME/ibm/
+export PATH=$GOPATH/bin/:$PATH:/usr/local/go/bin/
+
+# Repository Path
+mkdir -p $GOPATH/bin/
+mkdir -p $GOPATH/src/
+mkdir -p $GOPATH/pkg/
+mkdir -p $GOPATH/src/github.com/3scale/
+cd $GOPATH/src/github.com/3scale/
+
+
+# Install & Setup dep
+curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+git clone https://github.com/3scale/3scale-istio-adapter -b code-cleanups
```

Comment:
Remarks from Amir - 
`branch="${1:-master}`

make use of $1 and the default value of $1 will be master unless specified otherwise.